app/furniture/journal/entry_component.rb

```diff
@@ -9,10 +9,29 @@ def initialize(*args, entry:, **kwargs)
     end
 
     def body_html
-      render_markdown(entry.body)
+      postprocess(render_markdown(entry.body))
     end
 
-    def published_at
+    private def postprocess(text)
+      entry.keywords.map do |keyword|
+        replace_keyword(text, keyword)
+      end
+
+      text
+    end
+
+    private def replace_keyword(text, keyword)
```

Comment:
Other name suggestions my brain comes up with: 
- replace_keywords_with_link
- keywords_to_link
- linkify_keywords

☺️

Reply:
I like `linkify_keywords`! Wil try to do that.